sycl/source/platform.cpp

```diff
@@ -21,7 +21,7 @@ platform::platform() : impl(std::make_shared<detail::platform_impl>()) {}
 platform::platform(cl_platform_id PlatformId)
     : impl(std::make_shared<detail::platform_impl>(
           detail::pi::cast<detail::RT::PiPlatform>(PlatformId),
-          RT::GlobalPlugin)) {}
+          RT::getPlugin<backend::opencl>())) {}
```

Comment:
Just curious.

> Currently SYCL object creation from OpenCL object fails with segmentation fault because of invalid GlobalPlugin

`RT::GlobalPlugin` is a static global variable which is a pointer (I believe, an atomic one) to initialized plugin. Now, `RT::getPlugin<...>()` initializes this pointer.
Am I correct?

Reply:
That's right.